apriltag/src/main/java/edu/wpi/first/apriltag/AprilTagFieldLayout.java

```diff
@@ -221,6 +222,22 @@ public void serialize(Path path) throws IOException {
     new ObjectMapper().writeValue(path.toFile(), this);
   }
 
+  /**
+   * Get a standard {@link AprilTagFieldLayout}.
```

Comment:
Standard in what way?

Reply:
As in prebuilt/official- I'm not sure how else to indicate layouts with a corresponding `AprilTagFields` enum member. Maybe official would be better? `loadFromResource()` uses that wording, so at least it'd be consistent. (The C++ `LoadAprilTagLayoutField()` uses predefined, so that's also an option)

Reply:
I like "official".